tests/test_query.py

```diff
@@ -575,13 +576,38 @@ def test_query_category(client):
     response_data = res.json()["data"]
     assert len(response_data) == 1
     assert "idp" in response_data[0]
+    assert "ip" in response_data[0]
     assert "guid" not in response_data[0]
 
     # query a category that doesn't exist
     res = client.get("/log/whatisthis", headers={"Authorization": f"bearer {fake_jwt}"})
     assert res.status_code == 400, res.text
 
 
+def test_query_login_filter_by_ip(client):
+    # submit a login audit log
```

Comment:
to test that filtering is working, you would need to submit multiple login logs with different IPs.
But i would argue this test is not necessary, we already have `test_query_field_filter` that tests filtering logs.